src/hometask5/max_divisor.py

```diff
@@ -0,0 +1,27 @@
+# Вводится число найти его максимальный делитель,
```

Comment:
it was task for binary shift ^)

Reply:
Made new solution, left and commented the old one